packages/toggle-switch-react/src/ToggleSwitch.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
```

Comment:
We should add a test for the onchange handler

Reply:
Done :)